### DISCUSSION TITLE
Copilot 1.73.3.0 inline suggestions do not work with Visual Studio 17.5

### DISCUSSION THREAD

**Discussion by JeremyTCD** [community/community#48352] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Versions

Visual Studio: 17.5.0
Copilot extension: 1.73.3.0

### Description
After upgrading VS to 17.5.0, inline suggestions stopped working. Both alt+\ and manually clicking "trigger inline suggestion" do not work.

The copilot tab does display suggestions when open.

Copilot output panel does not display any errors.

After rolling back VS to 17.4.5, inline suggestions started working again.

**Comment by iamcjwilliams** [comment]
@JeremyTCD Thank you for your feedback, our copilot team has been notified!
